src/js.rs

```diff
@@ -13,7 +13,7 @@ where
     let constructor_name = Object::get_prototype_of(value).constructor().name();
 
     if constructor_name == classname {
-        let pointer = Reflect::get(value, &JsValue::from_str("ptr"))
+        let pointer = Reflect::get(value, &JsValue::from_str("__wbg_ptr"))
```

Comment:
cf https://github.com/rustwasm/wasm-bindgen/pull/3408

Reply:
This seems rather brittle but I guess the wbg dependency is pinned anyways 🥲

Reply:
totally agree. 

We need to get rid of this function anyway, per https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/issues/51